tests/time_tests/common/main.cpp

```diff
@@ -27,12 +27,21 @@ bool parseAndCheckCommandLine(int argc, char **argv) {
 }
 
 
+/**
+* @brief Function calls `runPipeline` with mandatory time tracking of full run
```

Comment:
Brief only comments can be single line. Also @breif is redundant - the first line is the @brief.

/// Function calls `runPipeline` with mandatory time tracking of THE full run

Reply:
@asomsiko it was copied from Inference Engine source code (especially benchmark_app). If required, I will update